### PR TITLE
Translations confirmation modal

### PR DESCRIPTION
**Pull request by nihco2**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | Translation confirmation modal
| Type?         | new feature
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/FF-80
| How to test?  | Check behavior of page after edit and leave page

### CODE REVIEW

File: src/PrestaShopBundle/Translation/Api/InternationalApi.php [hunk 1]
```diff
@@ -36,7 +36,10 @@ public function getTranslations()
         return array(
             'button_reset' => $this->translator->trans('Reset', array(), 'Admin.Actions'),
             'button_save' => $this->translator->trans('Save', array(), 'Admin.Global'),
+            'button_leave' => $this->translator->trans('Leave Anyway', array(), 'Admin.Global'),
```
Comment (AlexEven) [inline src/PrestaShopBundle/Translation/Api/InternationalApi.php]: Could we have "Leave anyway" (no capital A) and domain "Admin.Notifications.Warning" ?

File: src/PrestaShopBundle/Translation/Api/InternationalApi.php [hunk 2]
```diff
@@ -36,7 +36,10 @@ public function getTranslations()
...
             'button_search' => $this->translator->trans('Search', array(), 'Admin.Actions'),
+            'modal_title' => $this->translator->trans('Confirm this action', array(), 'Admin.Global'),
```
Comment (AlexEven) [inline src/PrestaShopBundle/Translation/Api/InternationalApi.php]: domain "Admin.Actions"

File: src/PrestaShopBundle/Translation/Api/InternationalApi.php [hunk 3]
```diff
@@ -36,7 +36,10 @@ public function getTranslations()
...
+            'modal_content' => $this->translator->trans('Your modifications are not saved yet. Do you wish to save it before leaving?', array(), 'Admin.Global'),
```
Comment (AlexEven) [inline src/PrestaShopBundle/Translation/Api/InternationalApi.php]: domain "Admin.Notifications.Warning"

File: src/PrestaShopBundle/Translation/Api/InternationalApi.php (hunk 1 above)
Comment (nihco2) [inline src/PrestaShopBundle/Translation/Api/InternationalApi.php]: sure

**Review by AlexEven — COMMENTED** [review]
Please see my comments :)
Thanks!

**Review by maximebiloe — APPROVED** [review]

**Comment by maximebiloe** [comment]
Thank you @nihco2

**Pull request merged**